stand/libsa/ufs.c

```diff
@@ -207,10 +205,7 @@ read_inode(inumber, f)
  * contains that block.
  */
 static int
-block_map(f, file_block, disk_block_p)
-	struct open_file *f;
-	ufs2_daddr_t file_block;
-	ufs2_daddr_t *disk_block_p;	/* out */
+block_map(struct open_file *f, ufs2_daddr_t file_block, ufs2_daddr_t *disk_block_p)	/* out */
```

Comment:
This line may now be > 80 characters. Also the comment at the end doesn't make sense. It was bound to the disk_block_p function argument (it's output by block_map).